tests/unit/test_masks.py

```diff
@@ -1,7 +1,11 @@
 import numpy as np
 import pytest
 
-from histolab.filters.compositions import _SlideFiltersComposition
+from histolab.filters import image_filters
```

Comment:
please could avoid importing the whole module? I'd like to see the import statements only for the object needed 😄